lib/javascript/food/foodForm.js

```diff
@@ -0,0 +1,41 @@
+const html = require('./html')
+
+class FoodForm {
+  constructor(){}
+
+  renderFoodForm() {
+    $('.new-food-form-container').after(html.foodForm())
+  }
+
+  listen() {
+    this.listenForSubmit()
+  }
+
+  listenForSubmit(){
+    $('.new-food-form').on('submit', function(){
+      event.preventDefault();
+      var foodName = $('input[name="new-food-name"]').val();
+      var foodCalories = $('input[name="new-food-calories"]').val();
```

Comment:
could you potentially extract grabbing these values out of this function. should grabbing the data be within the `listenForSubmit` function if we think about SRP?